lib/src/policy/api.ts

```diff
@@ -1,10 +1,13 @@
-import { NetworkError, ServiceError } from '../errors.js';
+import { NetworkError } from '../errors.js';
 import { AuthProvider } from '../auth/auth.js';
-import { rstrip } from '../utils.js';
-import { GetAttributeValuesByFqnsResponse, Value } from './attributes.js';
+import { extractRpcErrorMessage, rstrip } from '../utils.js';
+import { PlatformClient } from '../platform.js';
+import { Value } from './attributes.js';
+import { GetAttributeValuesByFqnsResponse } from '../platform/policy/attributes/attributes_pb.js';
 
+// TODO KAS: go over web-sdk and remove policyEndpoint that is only defined to be used here
```

Comment:
We are reaching a point where policyEndpoint can be replaced by platformUrl, the update will require fixing xtests afaik

Reply:
Maybe if we leave `policyEndpoint` in the `cli` no update to x-tests will be needed